### PR TITLE
Add Eth2Dai reserve

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: contractsV5/KyberReserveInterfaceV5.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.5.11;
+
+import "./IERC20Interface.sol";
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/KyberReserveInterfaceV5.sol]: Changed

File: contractsV5/IERC20Interface.sol [hunk 2]
```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.5.11;
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/IERC20Interface.sol]: Updated

File: contractsV5/KyberReserveInterfaceV5.sol [hunk 3]
```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.5.11;
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/KyberReserveInterfaceV5.sol]: Updated

File: contractsV5/eth2dai/Eth2DaiReserve.sol [hunk 4]
```diff
@@ -0,0 +1,839 @@
+pragma solidity 0.5.11;
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/eth2dai/Eth2DaiReserve.sol]: Updated folder structures

File: contractsV5/eth2dai/OtcInterfaceV5.sol [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/eth2dai/OtcInterfaceV5.sol]: Removed

File: contractsV5/eth2dai/OtcInterfaceV5.sol [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+pragma solidity 0.5.11;
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/eth2dai/OtcInterfaceV5.sol]: Updated

File: contractsV5/eth2dai/mock/TestTokenV5.sol [hunk 7]
```diff
@@ -0,0 +1,200 @@
+pragma solidity 0.5.11;
+
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply contractsV5/eth2dai/mock/TestTokenV5.sol]: Updated

**Comment by ilanDoron** [comment]
this PR was done in another one.

**Pull request closed without merging**